tests/lms/conftest.py

```diff
@@ -224,7 +224,12 @@ def lti_launch_request(monkeypatch, pyramid_request):
         "pylti.common.verify_request_common", lambda a, b, c, d, e: True
     )
 
-    pyramid_request.context = {"rpc_server_config": {}, "hypothesis_config": {}}
+    pyramid_request.context = mock.MagicMock(
+        spec_set=["rpc_server_config", "hypothesis_config", "h_pubid"],
```

Comment:
Is this `spec_set` going to be OK over time or do we risk it getting out of sync with the context class?